gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/GradleUtil.java

```diff
@@ -180,7 +180,7 @@ private static File findClassesOutputDirectory(Project gradleProject) {
   }
 
   private static File findArtifact(Project gradleProject) {
-    return new ArrayList<Configuration>(gradleProject.getConfigurations()).stream()
+    return new ArrayList<>(gradleProject.getConfigurations()).stream()
```

Comment:
isn't it generating a warning in the IDE or when compiling ?

Reply:
I don't understand. Are you referring to my usage of the diamond operator? While compiling kubernetes gradle plugin I see following warning:
```sh
kubernetes : $ mvn clean install
[INFO] Scanning for projects...
[WARNING] The project org.eclipse.jkube.kubernetes:org.eclipse.jkube.kubernetes.gradle.plugin:jar:1.6.0-SNAPSHOT uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin. See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html

...
[WARNING] bootstrap class path not set in conjunction with -source 8

```